### DISCUSSION TITLE
Change footer logo (Not Hide)

### DISCUSSION THREAD

**Discussion by mstfkhazaal** [filamentphp/filament#6091] [Help] [answered]
I want to put logo and links in the footer alternative filament logo, How i do it?

**Comment by danharrin** [comment, marked as answer]
Override the `footer.blade.php` view like you would another Laravel view

**Comment by marcosdsdba** [comment]
Try this:
https://laraveldaily.com/post/filament-add-theme-footer-with-render-hooks
